openstack/blockstorage/extensions/backups/requests.go

```diff
@@ -213,3 +213,35 @@ func RestoreFromBackup(client *gophercloud.ServiceClient, id string, opts Restor
 	})
 	return
 }
+
+// Export will export a Backup information. To extract the Backup export record
+// object from the response, call the Extract method on the ExportResult.
+func Export(client *gophercloud.ServiceClient, id string) (r ExportResult) {
+	_, r.Err = client.Get(exportURL(client, id), &r.Body, nil)
+	return
+}
+
+// ImportOpts contains options for importing a Backup. This object is passed to
+// the backups.ImportBackup function.
+type ImportOpts BackupRecord
+
+// ToImportMap assembles a request body based on the contents of a
+// ImportOpts.
+func (opts ImportOpts) ToImportMap() (map[string]interface{}, error) {
```

Comment:
nit: `ToBackupImportMap()`. I forget if I mentioned in the previous PR that the naming was OK because `ToBackupBackup` would have been really awkward.

Reply:
Do I also need to rename `Export` -> `BackupExport` and `Import` -> `BackupImport`?

Reply:
The function names? No - those can stay.

It's just the interface method name that needs to change to `To-Resource/Package-Action-ResultType` If you browse the other packages, you'll see that the interface names are verbose like this while the function/action names can be simple verbs.

I believe the reason behind this naming convention was to ensure that there is never a conflict with implementing a custom struct that can do many different things (implement many different interfaces).

Reply:
Ok, I'll rename only this method name.